### DISCUSSION TITLE
Setup Null-ls with debug mode

### DISCUSSION THREAD

**Discussion by bncpr** [LunarVim/LunarVim#2704] [Q&A] [answered]
Is there a way I can configure null-ls with debug = true to have logs?

**Comment by KalleChen** [comment]
In my case, I set this line `require("null-ls").setup({debug = true})` in `config.lua`, then I can get the log in `~/.cache/nvim/null-ls.log`

**Comment by LostInTheLogs** [comment, marked as answer]
`lvim.lsp.null_ls.setup.debug = true`

**Comment by mtzfactory** [comment]
Once the debug mode is enabled, how can I see the logs?

**Reply by LostInTheLogs** [reply]
`:NullLsLog` iirc, and fyi lunarvim is not currently maintained
